app/assets/javascripts/users.js

```diff
@@ -5,6 +5,16 @@ function hideField(){
 	document.getElementById("hidden").style.display = 'none';
 }
 
+function showCreateButton(){
+	if(document.getElementById("fake-signup-button").style.display != 'none'){

```

Comment:
You can use a variable here to avoid looking up the element every time

Reply:
done